src/load/azext_load/data_plane/utils/validators.py

```diff
@@ -0,0 +1,335 @@
+            raise InvalidArgumentValueError(
+                f"Provided path '{path}' is not writable or executable"
+            )
+    else:
+        if not os.path.isfile(path):
+            raise InvalidArgumentValueError(f"Provided path '{path}' is not a file")
+        if not os.access(path, os.R_OK):
+            raise InvalidArgumentValueError(f"Provided path '{path}' is not readable")
+    return path
+
+
+def validate_file_type(namespace):
+    if namespace.file_type is None:
+        return
+    if not isinstance(namespace.file_type, str):
+        raise InvalidArgumentValueError(
+            f"Invalid file-type type: {type(namespace.file_type)}"
+        )
+    allowed_file_types = utils.get_enum_values(AllowedFileTypes)
+    if namespace.file_type not in allowed_file_types:
+        raise InvalidArgumentValueError(
+            f"Invalid file-type value: {namespace.file_type}. Allowed values: {', '.join(allowed_file_types)}"
+        )
+
+
+def validate_start_iso_time(namespace):
+    _validate_iso_time(namespace.start_time)
+
+
+def validate_end_iso_time(namespace):
+    _validate_iso_time(namespace.end_time)
+
+
+def _validate_iso_time(string):
+    if string is None:
+        return
+    if not isinstance(string, str):
+        raise InvalidArgumentValueError(f"Invalid time type: {type(string)}")
+    try:
+        datetime.strptime(string, "%Y-%m-%dT%H:%M:%S.%fZ")
+        return
+    except ValueError:
+        pass
+    try:
+        datetime.strptime(string, "%Y-%m-%dT%H:%M:%SZ")
+        return
+    except ValueError:
+        pass
+    raise InvalidArgumentValueError(
+        f"Invalid time format: '{string}'. Expected ISO 8601 format."
+    )
+
+
+def validate_interval(namespace):
+    if namespace.interval is None:
+        return
+    if not isinstance(namespace.interval, str):
+        raise InvalidArgumentValueError(
+            f"Invalid interval type: {type(namespace.interval)}"
+        )
+    allowed_intervals = utils.get_enum_values(AllowedIntervals)
+    if namespace.interval not in allowed_intervals:
+        raise InvalidArgumentValueError(
+            f"Invalid interval value: {namespace.interval}. Allowed values: {', '.join(allowed_intervals)}"
+        )
+
+
+def validate_metric_namespaces(namespace):
+    if not isinstance(namespace.metric_namespace, str):
+        raise InvalidArgumentValueError(
+            f"Invalid metric-namespace type: {type(namespace.metric_namespace)}"
+        )
+    allowed_metric_namespaces = utils.get_enum_values(AllowedMetricNamespaces)
+    if namespace.metric_namespace not in allowed_metric_namespaces:
+        raise InvalidArgumentValueError(
+            f"Invalid metric-namespace value: {namespace.metric_namespace}. "
+            f"Allowed values: {', '.join(allowed_metric_namespaces)}"
+        )
+
+
+def validate_dimension_filters(namespace):
+    """Extracts multiple space and comma-separated dimension filters in key1[=value1,value2] format"""
```

Comment:
Usually, we use `key1=value1 key1=value2` to pass multiple values.

Reply:
The difference is that we use space to separate two equations __key1=value` `key1=value__. Please remove the second case.

Reply:
We have updated the implementation